pkg/cache/cache.go

```diff
@@ -0,0 +1,158 @@
+package cache
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+)
+
+var DefaultCacheExpiration = time.Now().Add(30 * time.Minute)
+
+const (
+	RosaConfigDir = "ROSA_CONFIG_DIR"
```

Comment:
@ciaranRoche Just wondering if we should reuse `OCM_CONFIG` environment variable or do you see value in having a separate `ROSA_CONFIG_DIR` variable? I wasn't aware of the `OCM_CONFIG` variable until taking a look at the `rosa edit/get config` commands.

Reply:
Currently uses the OCM Config, so either we stick with that and use the OCM_CONFIG or we create a specific rosa config. I am leading on the former, what do you think?
Both have pro's and cons

Reply:
@ciaranRoche Coming back to this, I think we should use `OCM_CONFIG` for the time being rather than introducing another variable just for this.